contracts/staking/AbstractGoodStaking.sol

```diff
@@ -82,13 +82,11 @@ contract AbstractGoodStaking {
      * @return Intrest gained on lending the tokens.
      * @return Intrest gained on lending the tokens in terms of token rate.
      * @return Token's precision loss due to decimal difference.
-     * @return average intrest donation ratio.
```

Comment:
we said not to remove it, to keep it working with current staking contract

Reply:
added back